modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2703,6 +2703,28 @@ def explode(self, axis: Union[int, Axis], func: Callable) -> "PandasDataframe":
             partitions, new_index, new_columns, row_lengths, column_widths
         )
 
+    def to_pandas_in_remote_function(self) -> "PandasDataframe":
+        """
+        Create a PandasFrame with single partition from the partitions of the current dataframe.
+
+        Returns
+        -------
+        PandasDataframe
+            An PandasDataframe containing only a single partition.
+        """
+        partition_with_to_pandas_data = [
+            self._partition_mgr_cls.convert_partitions_to_pandas_in_remote_func(
+                self._partitions
+            )
+        ]
+        partitions = np.array(partition_with_to_pandas_data).reshape(1, -1)
+
+        result = self.__constructor__(
+            partitions, self.index, columns=self.columns, dtypes=self.dtypes
```

Comment:
What about row_lengths and column_widths?

Reply:
As the partitions are different for the new dataframe the row_lengths and col_widths would not be same as self

Reply:
We could sum them up if they are available.

```python
new_row_lengths = [sum(self._row_lengths_cache)]
new_column_widths = [sum(self._colum_widths_cache)]
```

Reply:
This should help